portfolio/src/main/webapp/scripts/controllers/comments_controller.js

```diff
@@ -67,4 +67,21 @@ slider.oninput = function() {
   renderCommentList(commentList.slice(0, this.value));
 };
 
-export { requestComments };
\ No newline at end of file
+const addNewCommentToList = newComment => {
+  commentList.unshift(newComment);
+  setSliderProps();
+  formatCommentHeaderText(slider.value, commentList.length);
+  renderCommentList(commentList.slice(0, slider.value));
+}
+
+const url = 'https://socket-dot-scarletnguyen-step-2020.uk.r.appspot.com/';
+const socket = io.connect(url);
```

Comment:
I'm not entirely familiar with this but what happens after 60 seconds?  Do you have to reconnect?

Reply:
Yes, after 60 seconds of being idle, the connect will be automatically close until the next API call.